Symfony/CS/Fixer/Contrib/ClassKeywordRemoveFixer.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class ClassKeywordRemoveFixer extends AbstractFixer
+{
+    /**
+     * @var string[]
+     */
+    private $imports = array();
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        $this->storeImports($tokens);
+        $tokens->rewind();
+        $this->replaceClassKeywords($tokens);
+
+        return $tokens->generateCode();
+    }
+
+    /**
+     * @param Tokens $tokens
+     */
+    private function storeImports(Tokens $tokens)
+    {
+        foreach ($tokens->getImportUseIndexes() as $index) {
+            $import = '';
+            while (($index = $tokens->getNextMeaningfulToken($index))) {
+                if ($tokens[$index]->equals(';')) {
```

Comment:
Indeed :)
But please test something like as well, these are class uses
```php
use some\test\{ClassA, ClassB, ClassC as C};
use some\test\{ClassD, ClassE, ClassF}?>
```

Thanks for adding the test, however you need a little PHP7 req. wrapper for when the test are run on < PHP7

Example;
```php
/**
     * @dataProvider provide70Cases
     * @requires PHP 7.0
     */
    public function test70($expected, $input = null)
    {
         $this->makeTest($expected, $input);
    }
 
    public function provide70Cases()
    {
        return array(
            array()
        );
    }
```